include_core/omrutil.h

```diff
@@ -300,6 +300,21 @@ omr_error_t detectVMDirectory(wchar_t *vmDirectory, size_t vmDirectoryLength, wc
 
 uintptr_t getStorageKey(void);
 
+/* ---------------- zosversion.c ---------------- */
+
+#define ZOS_V2R4_RELEASE 27.00
+#define ZOS_V2R4_VERSION 4
+
+/**
+ * Function to determine if the zos version is at least a given
+ * release and version.  The implementation is based on uname(),
+ * NOT on __osname() as the __osname() release numbers are not
+ * guaranteed to increase.
+ *
+ * For release and version numbers, see
+ * 	https://www.ibm.com/docs/en/zos/2.5.0?topic=functions-uname-display-current-operating-system-name
+ */
+BOOLEAN zos_version_at_least(double min_release, double min_version);
 #endif /*if defined(J9ZOS390)*/
```

Comment:
While you're here, please correct this comment - it should repeat the `#if` condition:
```c
#endif /* defined(J9ZOS390) */
```